README.md

```diff
@@ -5,84 +5,41 @@
+impossible to quickly identify. Defenders can use BloodHound to identify and eliminate those same attack paths. Both blue and red
+teams can use BloodHound to easily gain a deeper understanding of privilege relationships in an Active Directory or Azure
 environment.
 
 BloodHound CE is created and maintained by the [BloodHound Enterprise Team](https://bloodhoundenterprise.io). The
-original
-BloodHound was created by [@_wald0](https://www.twitter.com/_wald0), [@CptJesus](https://twitter.com/CptJesus), and
+original BloodHound was created by [@_wald0](https://www.twitter.com/_wald0), [@CptJesus](https://twitter.com/CptJesus), and
 [@harmj0y](https://twitter.com/harmj0y).
 
-## Quick Start
-
-BloodHound CE is distributed as a Docker image available at https://hub.docker.com/r/specterops/bloodhound.
-In order to get started, an example docker compose folder is provided
-at [examples/docker-compose](examples/docker-compose/README.md).
-
-### Prerequisites
-
-Running the example Docker Compose project requires the following:
-
-- [Docker](https://www.docker.com/)
-- [Docker Compose](https://docs.docker.com/compose/)
-
-The simplest way to get started is to install Docker Desktop, as it will provide both prerequisites and requires no
-additional
-configuration: [Docker Desktop](https://www.docker.com/products/docker-desktop/)
-
-Because BloodHound CE is distributed as a Docker image, there are many other ways to deploy, but this provides the
-simplest setup.
-
-### Running with Docker Compose
-
-- Download our example [docker-compose.yml](./examples/docker-compose/docker-compose.yml)
-- In whatever directory you copied the `docker-compose.yml` file to, run `docker compose up`
-- The initial password will show up in the logs when the API server fully starts (there's a "server started" message
-  that will appear)
-- Go to `http://localhost:8080/ui/login` and use the username `admin` with your randomly generated password
-- Congrats, you're now running BloodHound and can complete your application setup
-
-### Helpful Tips
-
-- You'll be asked to change your password on first login
-- To get the latest compatible collectors, simply click the gear icon in the corner and select "Download Collectors"
-- If you restart the service before copying your random password, it will not be regenerated. Simply
-  run `docker compose down -v`
-  and then `docker compose up` to reset your databases.
-- More information, troubleshooting, and how to configure your deployments can be found
-  in [Docker Compose Example README](./examples/docker-compose/README.md)
+## Running BloodHound Community Edition
 
+The easiest way to get up and running is to use our pre-configured Docker Compose setup. The following steps will get BloodHound CE up and running with the least amount of effort.
+  
+  1. Install [Docker Compose](https://docs.docker.com/compose/install/)
```

Comment:
These numbers are all 1

Reply:
They display correctly when viewing the rendered markdown. For some reason I thought this was the preferred way of doing numbered lists, but some quick searches say otherwise. I'll fix them up to be more current.